packages/extension-browser/src/devtools/views/controls/label-text.css

```diff
@@ -0,0 +1,4 @@
+.root {
```

Comment:
This control should be able to be removed once we're using grid for the label control.